components/example-app/extensions/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "example-extension",
+  "version": "0.1.0",
+  "description": "",
+  "license": "MIT",
+  "main": "Example.jsx",
+  "scripts": {
+    "dev": "hs-ui-extensions-dev-server dev",
+    "build": "hs-ui-extensions-dev-server build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/HubSpot/ui-extensions-react-examples"
```

Comment:
Debating about whether to change the link here--it could be nice to direct customers to the repo with the full example project, but I wonder whether that might be confusing.

Reply:
Hmm I think the `repo` field is meant to be where this specific repo lives on GH so this seems ok as is

Reply:
This extensions file seems to contain a lot more than what's contained in the [current package.json for the app in the getting started project](https://github.com/HubSpot/hubspot-project-components/blob/main/projects/getting-started-template/src/app/extensions/package.json).